### PR TITLE
CC-24698: Adjusted product feature guide for discontinued variant postfix integration description.

### PR DESCRIPTION
**Pull request by sakharova-yuliia**
Ticket: https://spryker.atlassian.net/browse/CC-24698

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-product-feature.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
+
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202304.0/install-the-product-feature.md:1]: please use the corresponding doc from 202212.0 as a reference and add the new functionality (the document will have different structure so I haven't reviewed this one in details). Recheck the new structure and place plugin and config according to a new structure to the proper section

File: _includes/pbc/all/install-features/202304.0/install-the-product-feature.md [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+### 4) Set up behaviour
+
+1. Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                       | SPECIFICATION                                                                                  | PREREQUISITES | NAMESPACE                                                       |
+|----------------------------------------------|------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------|
+| ProductViewDiscontinuedOptionsExpanderPlugin | Expands the product view super attributes and attribute variant map with discontinued postfix. |               | Spryker/Client/ProductDiscontinuedStorage/Plugin/ProductStorage |
+
+**\Pyz\Zed\Oms\OmsDependencyProvider.php**
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202304.0/install-the-product-feature.md:118]: not OMS

File: _includes/pbc/all/install-features/202304.0/install-the-product-feature.md [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+    protected function getProductViewExpanderPlugins(): array
+    {
+        $plugins = [
+            new ProductViewDiscontinuedOptionsExpanderPlugin(),
+        ];
+
+        return $plugins;
+    }
+}
+```
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202304.0/install-the-product-feature.md:142]: verification

File: _includes/pbc/all/install-features/202304.0/install-the-product-feature.md [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+
+{% endinfo_block %}
+
+### 4) Set up behaviour
+
+1. Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                       | SPECIFICATION                                                                                  | PREREQUISITES | NAMESPACE                                                       |
+|----------------------------------------------|------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------|
+| ProductViewDiscontinuedOptionsExpanderPlugin | Expands the product view super attributes and attribute variant map with discontinued postfix. |               | Spryker/Client/ProductDiscontinuedStorage/Plugin/ProductStorage |
```
Comment (ilyakubanov) [inline _includes/pbc/all/install-features/202304.0/install-the-product-feature.md:116]: where is it taken from? specification from a corresponding client method can be used

**Review by ilyakubanov — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-product-feature.md (hunk 4 above)
Comment (sakharova-yuliia) [reply _includes/pbc/all/install-features/202304.0/install-the-product-feature.md]: Taken from the plugin specification

**Comment by ilyakubanov** [comment]
Will be done in https://spryker.atlassian.net/browse/CC-25452

**Pull request closed without merging**